### DISCUSSION TITLE
Installation in disconnected environment

### DISCUSSION THREAD

**Discussion by Hapshanko** [ansibleguy76/ansibleforms#159] [Announcements]
Hi! Was looking to try out AnsibleForms in our disconnected on prem environment.
I read that the docker-compose searches for the latest image of AnsibleForms in docker hub, since docker hub will not be accessible, is there a way I can define this image to be pulled from my own image registry?

**Comment by ansibleguy76** [comment]
that's probably more a question for a docker forum.  i assume there are a few ways to that?  in the end it just looks for the local image.
